blueprints/flexible-topologies.md

```diff
@@ -405,6 +405,8 @@ Once all Delivery Services have been migrated to Topologies, CDN operators will
 
 Troubleshooting issues with Delivery Services on the CDN may become slightly more difficult due to the fact that you will no longer be able to assume that any given Delivery Service will follow the same global hierarchy. Unique Topologies will provide unique paths through the CDN from client to origin, so an operator now needs to look up what the path *should be* based on the Delivery Service's Topology in order to triage issues along the delivery path. This may necessitate the development of Topology-based troubleshooting tools and visualizations.
 
+As something to be aware of, content revalidations (via `regex_revalidate.config`) may take longer to propagate through a Topology the more tiers that it has, due to the nature of how ORT waits to revalidate until its cache's parents no longer have revalidations pending.
```

Comment:
This is actually a big problem if you have revalidations coming in faster than they can chain through your topology since it's a global state on the server object.  I also don't know how any of that would work if you have multiple topologies for a given server with different parentages.

Reply:
Since revalidations are basically per-CDN at this point, a given server would have to wait for all of its parents (via all of the Topologies it is part of). You do bring up a good point about revalidations coming in too fast -- if that becomes a problem we will probably have to come up with a new process where each revalidation has a timestamp associated with it so that revalidations can be done up to a certain point in time rather than needing to wait until everything is caught up. Although that would be out of the scope of this blueprint -- we can cross that bridge when it comes :)

Reply:
If you're measuring revalidation time based on how long it takes to fully propagate to the client edge today it's per CDN * number of tiers * revalidation interval.  I raise it mostly because of the scalability concern that as the number of tiers increases, the total time will also increase.  Additionally, the probability that it will be pre-empted by the submission of another job also increases.  If we want to just kick that can down the road, we can.